src/common/LoggedInProviders.tsx

```diff
@@ -31,7 +62,6 @@ export const LoggedInProviders = ({
                   </NotificationsManagerProvider>
                 </PushNotificationsProvider>
                 <CreateEditPostModal />
-                <Toast />
```

Comment:
This Toast had to move so that the "No Internet" message could still be shown on the LoginScreen.